source/stylesheets/refills/_side-image.scss

```diff
@@ -1,23 +1,11 @@
-.side-image {  
-  ///////////////////////////////////////////////////////////////////////////////////
-  $base-line-height: 1.5em !default;
-  $base-accent-color: #477DCA !default;
-  $base-link-color: $base-accent-color !default;
-  $dark-gray: #333 !default;
-  $large-screen: em(860) !default;
-  $base-font-color: $dark-gray !default;
-  
-  p {
-    color: $base-font-color;
-    line-height: $base-line-height;
-  }
-  //////////////////////////////////////////////////////////////////////////////////
-
+.side-image {
+  $side-image-color: $base-font-color-on-box;
```

Comment:
Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets